apps/wing/src/commands/test.ts

```diff
@@ -153,6 +158,65 @@ async function testSimulator(synthDir: string) {
   }
 }
 
+async function testAwsCdk(synthDir: string): Promise<sdk.cloud.TestResult[]> {
+
+  if (!isAwsCdkInstalled(synthDir)) {
+    throw new Error(
+      "AWS-CDK is not installed. Please install AWS-CDK to run tests in the cloud (npm i aws-cdk)."
```

Comment:
My bad -- I think it only works when it is globally installed (otherwise aws-cdk will not be on your PATH), so let's recommend that instead

```suggestion
      "AWS-CDK is not installed. Please install AWS-CDK to run tests in the cloud (npm i -g aws-cdk)."
```